packages/components/src/hooks/useStore.test.ts

```diff
@@ -0,0 +1,74 @@
+import { Atom, atom } from '@terra-ui-packages/core-utils';
+import { act, renderHook } from '@testing-library/react';
+
+import { useSettableStore, useStore } from './useStore';
+
+describe('useStore', () => {
+  it('returns value from store', () => {
+    // Act
+    const store = atom<string>('foo');
+    const { result: hookReturnRef } = renderHook(useStore, { initialProps: store });
+
+    // Assert
+    expect(hookReturnRef.current).toBe('foo');
+  });
+
+  it('handles update to store', () => {
+    // Arrange
+    const store = atom<string>('foo');
+    const { result: hookReturnRef } = renderHook(useStore, { initialProps: store });
+
+    // Act
+    act(() => {
+      store.set('bar');
+    });
+
+    // Assert
+    expect(hookReturnRef.current).toBe('bar');
+  });
+});
+
+describe('useSettableStore', () => {
+  it('handles direct value update', () => {
+    // Arrange
+    const myAtom = atom<string>('hello');
+    const { result: hookReturnRef, rerender } = renderHook((args: [Atom<string>]) => useSettableStore(...args), {
+      initialProps: [myAtom],
+    });
+    const [initialValue, setValue] = hookReturnRef.current;
+
+    // Act
+    act(() => {
+      setValue('goodbye');
+    });
+    rerender([myAtom]);
```

Comment:
Why do you need to rerender? The documentation says that the component will rerender when the store changes.

Reply:
This was an existing test and I didn't look at it too closely when I copied it. But yeah, this looks like it invalidates the test.

Reply:
Fixed in https://github.com/DataBiosphere/terra-ui/pull/4805/commits/da23b4c556f50e70f5d754ab0f28aed7207f431c.